.gitignore

```diff
@@ -22,6 +22,4 @@ __mocks__/**/*.js
 /.templates
 template.config.js
 .env
-keys-whitelist.json
-keys-finance.json
```

Comment:
does a keys-finance.json file need to be added or is that a relic (and presumably nglFinanceBot in keys-whitelist)?

Reply:
correct, it was no longer referenced.